gcalbot/gcalbot/handler.go

```diff
@@ -31,14 +31,17 @@ func NewHandler(
 	config *oauth2.Config,
 	httpPrefix string,
 ) *Handler {
-	return &Handler{
+	handler := &Handler{
 		DebugOutput: base.NewDebugOutput("Handler", kbc),
 		kbc:         kbc,
 		db:          db,
 		requests:    requests,
 		config:      config,
 		httpPrefix:  httpPrefix,
 	}
+	// begin renewing old channels every hour
+	go handler.runRenewChannelScheduler()
```

Comment:
This should stop on shutdown, can you work something into `HandleSignals` so this can cleanup?